src/js/builtins/ImportMetaObject.ts

```diff
@@ -1,6 +1,6 @@
 type ImportMetaObject = Partial<ImportMeta>;
 
-$visibility = "Private";
+$visibility = "PrivateRecursive";
 export function loadCJS2ESM(this: ImportMetaObject, resolvedSpecifier: string) {
```

Comment:
Protects call stack of internals recursively. (see https://github.com/WebKit/WebKit/commit/37f814d56d6446ea011af18ee65b6644c1127b4e).